<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Proxima Nova</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/logg.qrc">
    <normaloff>:/ico2.png</normaloff>:/ico2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
color:#4fe3c3;
border:1px solid #325154;
}

QPushButton:hover{
background-color:#4fe3c3;
color:#262d38;
border:1px solid #262d38;
}

QLabel{
color:#b9c8c8;
}

QWidget{
background-color:#12161f;
}

QSlider::handle {
background-color:#325154;
border:0px solid #262d38;
}

QComboBox {
color:#4fe3c3;
border:1px solid #325154;
font-size:10pt;
font-family: &quot;Proxima Nova Rg&quot;;
}

QComboBox QAbstractItemView {
	color: #4fe3c3;
}

QStackedWidget {
border: 1px solid #325154;
}

font: 10pt &quot;Proxima Nova Rg&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="modelDropdown">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova Rg</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Choose Model</string>
        </property>
        <item>
         <property name="text">
          <string>Choose Model</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pre-trained model</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Self-trained model</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Self-trained model on faces</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="importImageBtn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Import image</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="importVideoBtn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Import video</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="colorizeImageBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Colorize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorizeVideoBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Colorize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loadingSpinner">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loadingText">
        <property name="font">
         <font>
          <family>Proxima Nova</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Colorizing...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QStackedWidget" name="input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="image_in">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="imgLabelIn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="VideoPlayer" name="video_in"/>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="output">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="image_out">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="imgLabelOut">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="VideoPlayer" name="video_out"/>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoPlayer</class>
   <extends>QWidget</extends>
   <header>VideoPlayer</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/logg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>importImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>importImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importVideoBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>importVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorizeVideoBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorizeVideoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorizeImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorizeImageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modelDropdown</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>getModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>importImage()</slot>
  <slot>colorizeImageClicked()</slot>
  <slot>importVideo()</slot>
  <slot>colorizeVideoClicked()</slot>
  <slot>play()</slot>
  <slot>setPosition()</slot>
  <slot>getModel()</slot>
 </slots>
</ui>
